src/components/EventForm/index.js

```diff
@@ -129,35 +137,73 @@ export default function EventForm({
       </div>
     </div>
   );
+  const accion=()=>{
+    onFinish(form.getFieldsValue());
+  }
+  
+  const onInput = (e) => {
+    //console.log(e.target.value)
+    setValueTitle(e.target.value)};
+
+  const onChange = (date, dateString) => {
+      //console.log(dateString);
+      if(dateString !== ""){
+        setValueDate(dateString)
+      }
+  };
+
 
   return (
     <Modal
       open={isModalOpen}
-      onOk={() => {
-        onFinish(form.getFieldsValue());
-      }}
+      onOk={()=>{accion}}
```

Comment:
accion is a function. Implementation here is wrong can you please revisit this @Blueclaus13 ?